splunktaucclib/rest_handler/handler.py

```diff
@@ -102,13 +102,39 @@ def check_existing(self, name):
             else:
                 return None
 
+        def basic_name_validation(name):
+            tmp_name = str(name)
+            prohibited_chars = ["*", "\\", "[", "]", "(", ")", "?", ":"]
+            val_err_msg = (
```

Comment:
Is nested tuple something necessary or it's done by accident?

Reply:
by accident, changed.